docs/setup.rst

```diff
@@ -0,0 +1,79 @@
+Setting Up Corobo
+=================
+
+The following documentation illustrates the process to install corobo and interface it with the gitter(or any other backend) API.
+
+Installation
+------------
+
+*  To install corobo, first you need to setup a virtual environment(it is not mandatory but highly recommended). This can be done as follows:
```

Comment:
```suggestion
*  To install corobo, first you need to setup a virtual environment. This can be done as follows:
```

Let's not involve non-venv setups, which most probably won't even work.